GammaJul.ReSharper.EnhancedTooltip/VisualStudio/QuickInfoSourceBase.cs

```diff
@@ -49,8 +49,9 @@ protected ITextBuffer TextBuffer
 			IDocument document = new JetVsTextBuffer(bufferAdapter).JetDocument.Value;
 			if (document == null)
 				return null;
-			
-			IDocumentMarkup documentMarkup = DocumentMarkupManagerBase.TryGetMarkupModel(document);
+
+
+		    IDocumentMarkup documentMarkup = Shell.Instance.GetComponent<IDocumentMarkupManager>().TryGetMarkupModel(document);
```

Comment:
Please use `Shell.Instance.TryGetComponent<IDocumentMarkupManager>()?.TryGetMarkupModel(document)` instead, I want `AugmentQuickInfoSession` do be safe (do nothing if we don't have everything we need) before calling `AugmentQuickInfoSessionCore`.